### PR TITLE
PoR developers guide

### PR DESCRIPTION
**Pull request by bayram98**
This PR is ready to review and merge.

### CODE REVIEW

File: en/developers-guide/proof-of-reserve.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+description: Ongoing proof of reserve validation for your financial entity.
+---
+
+# Proof of Reserve
+
+For a comprehensive illustration of utilizing **Orakl Network's Proof of Reserve**, refer to the 
+example repository of the [`data-feed-consumer`](https://github.com/Bisonai/data-feed-consumer). The on-chain implementation for Proof of Reserve mirrors that of the Aggregator on the Aggregator Proxy.
```
Comment (martinkersner) [inline en/developers-guide/proof-of-reserve.md:8]: Typo

```suggestion
example repository of the [`data-feed-consumer`](https://github.com/Bisonai/data-feed-consumer). The on-chain implementation for Proof of Reserve mirrors that of the Aggregator and the Aggregator Proxy.
```

File: en/developers-guide/proof-of-reserve.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## What is Proof of Reserve?
+
+The **Orakl Network's Proof of Reserve (PoR)** stands as a vital element in fostering trust and transparency within the financial ecosystem of 
+Orakl Network. Playing a pivotal role, Proof of Reserve is dedicated to the verification and validation of reserve holdings for financial
```
Comment (martinkersner) [inline en/developers-guide/proof-of-reserve.md:13]: Please do not mention "verification". PoR does not verify anything.

File: en/developers-guide/proof-of-reserve.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+# Proof of Reserve
+
+For a comprehensive illustration of utilizing **Orakl Network's Proof of Reserve**, refer to the 
+example repository of the [`data-feed-consumer`](https://github.com/Bisonai/data-feed-consumer). The on-chain implementation for Proof of Reserve mirrors that of the Aggregator on the Aggregator Proxy.
+
+## What is Proof of Reserve?
+
+The **Orakl Network's Proof of Reserve (PoR)** stands as a vital element in fostering trust and transparency within the financial ecosystem of 
+Orakl Network. Playing a pivotal role, Proof of Reserve is dedicated to the verification and validation of reserve holdings for financial 
+entities through a secure and auditable procedure.The Peg-Gold Proof of Reserve refers to a reserved quantity of non-fungible tokens (NFTs) issued specifically for PEG Gold.
```
Comment (martinkersner) [inline en/developers-guide/proof-of-reserve.md:14]:
```suggestion
entities through a secure and auditable procedure. The Peg-Gold Proof of Reserve refers to a reserved quantity of non-fungible tokens (NFTs) issued specifically for PEG Gold.
```

File: en/developers-guide/proof-of-reserve.md (hunk 2 above)
Comment (martinkersner) [reply en/developers-guide/proof-of-reserve.md:13]: At https://orakl.network/proof-of-reserve, we call it `GPC (Gold Pegged Coin)` and in here here we use `Peg-Gold` and `PEG Gold`. Please unify it according to the website if there is no reason behind using different names in here.

File: en/developers-guide/proof-of-reserve.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+# Proof of Reserve
+
+For a comprehensive illustration of utilizing **Orakl Network's Proof of Reserve**, refer to the 
+example repository of the [`data-feed-consumer`](https://github.com/Bisonai/data-feed-consumer). The on-chain implementation for Proof of Reserve mirrors that of the Aggregator on the Aggregator Proxy.
+
+## What is Proof of Reserve?
+
+The **Orakl Network's Proof of Reserve (PoR)** stands as a vital element in fostering trust and transparency within the financial ecosystem of
```
Comment (martinkersner) [inline en/developers-guide/proof-of-reserve.md:12]: Just curious, is there a reason to split sentences midway? 🤔

File: en/developers-guide/proof-of-reserve.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+### Supported Proof of Reserve on Cypress
+
+<table><thead><tr><th width="157">PoR</th><th>Aggregator</th><th>AggregatorProxy</th><th>Heartbeat (m)</th></tr></thead><tbody>
+    <tr>
+        <td>PEG-POR</td>
+        <td><a href="https://www.klaytnfinder.io/account/0xb5e91e5CE0B8e6fc3029b4E9ce057675a2c96dd1">0xb5e91e5CE0B8e6fc3029b4E9ce057675a2c96dd1</a></td>
+        <td><a href="https://www.klaytnfinder.io/account/0x9FbA23B10692cB3fa6Fea09834855ACc597BD180">0x9FbA23B10692cB3fa6Fea09834855ACc597BD180</a></td>
+        <td>60</td></tr></tr></tbody></table>
+
+### Supported Proof of Reserve on Baobab
```
Comment (martinkersner) [inline en/developers-guide/proof-of-reserve.md:26]: Please mention in here that the Baobab is using development API which does not have to return the same results as when used with production API used for Cypress deployment.

File: en/developers-guide/proof-of-reserve.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+The **Orakl Network's Proof of Reserve (PoR)** stands as a vital element in fostering trust and transparency within the financial ecosystem of 
+Orakl Network. Playing a pivotal role, Proof of Reserve is dedicated to the verification and validation of reserve holdings for financial 
+entities through a secure and auditable procedure.The Peg-Gold Proof of Reserve refers to a reserved quantity of non-fungible tokens (NFTs) issued specifically for PEG Gold.
+
+
+### Supported Proof of Reserve on Cypress
+
+<table><thead><tr><th width="157">PoR</th><th>Aggregator</th><th>AggregatorProxy</th><th>Heartbeat (m)</th></tr></thead><tbody>
+    <tr>
+        <td>PEG-POR</td>
```
Comment (martinkersner) [inline en/developers-guide/proof-of-reserve.md:21]: How about calling it as `GPC (Gold Pegged Coin)` as in https://orakl.network/proof-of-reserve?

File: en/developers-guide/proof-of-reserve.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+        <td><a href="https://www.klaytnfinder.io/account/0xb5e91e5CE0B8e6fc3029b4E9ce057675a2c96dd1">0xb5e91e5CE0B8e6fc3029b4E9ce057675a2c96dd1</a></td>
+        <td><a href="https://www.klaytnfinder.io/account/0x9FbA23B10692cB3fa6Fea09834855ACc597BD180">0x9FbA23B10692cB3fa6Fea09834855ACc597BD180</a></td>
+        <td>60</td></tr></tr></tbody></table>
+
+### Supported Proof of Reserve on Baobab
+
+<table><thead><tr><th width="157">PoR</th><th>Aggregator</th><th>AggregatorProxy</th><th>Heartbeat (m)</th></tr></thead><tbody>
+    <tr>
+    <td>PEG-POR</td><td><a href="https://www.klaytnfinder.io/account/0x58798D6Ca40480DF2FAd1b69939C3D29d91b60d3">0x58798D6Ca40480DF2FAd1b69939C3D29d91b60d3</a></td>
```
Comment (martinkersner) [inline en/developers-guide/proof-of-reserve.md:30]: The links are pointing to Cypress, not Baobab.

File: en/developers-guide/proof-of-reserve.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    <td><a href="https://www.klaytnfinder.io/account/0x821179a6d4F62fa6979BF42bEb9eE16a1F14C4eD">0x821179a6d4F62fa6979BF42bEb9eE16a1F14C4eD</a></td>
+    <td>2</td></tr></tr></tbody></table>
+
+</tbody></table>
+
+## How to read from Aggregator contract of PoR
+
+The Proof of Reserve (PoR) in Orakl Network is realized through two key smart contracts: Aggregator and AggregatorProxy. These contracts form a pair, representing a specific data feed, and enable continuous validation of reserve holdings. The Aggregator receives regular updates from off-chain oracles, ensuring accurate and up-to-date reserve information. The AggregatorProxy serves as a consistent API for accessing the submitted data.
```
Comment (martinkersner) [inline en/developers-guide/proof-of-reserve.md:38]: We should avoid mentioning verification/validation. Let's modify it as below.

```suggestion
The Proof of Reserve (PoR) in Orakl Network is realized through two key smart contracts: `Aggregator` and `AggregatorProxy`. These contracts form a pair, representing a specific data feed, and enable continuous access to the reserve holdings. The `Aggregator` receives regular updates from off-chain oracles, ensuring accurate and up-to-date reserve information. The `AggregatorProxy` serves as a consistent API for accessing the submitted data.
```

File: en/developers-guide/proof-of-reserve.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+For a detailed understanding of PoR and its integration, refer to the Orakl Network Data Feed documentation. This documentation provides comprehensive insights into the architecture, reading procedures, and the relationship between Aggregator and AggregatorProxy, enhancing your grasp of PoR functionality.
```
Comment (martinkersner) [inline en/developers-guide/proof-of-reserve.md:40]:
```suggestion
For a detailed understanding of PoR and its integration, refer to the Orakl Network Data Feed documentation. This documentation provides comprehensive insights into the architecture, reading procedures, and the relationship between `Aggregator` and `AggregatorProxy`, enhancing your grasp of PoR functionality.
```

**Review by martinkersner — APPROVED** [review]
I found some issues that we should fix before we merge. Please check them out and let me know if you need more detailed explanation.

**Review by nick-bisonai — APPROVED** [review]
lgtm!

**Review by martinkersner — APPROVED** [review]
LGTM!

**Pull request merged**